content/en/blog/_posts/2019-09-02-kubernetes-with-microk8s.md

```diff
@@ -0,0 +1,149 @@
+---                                           
+title: 'Running Kubernetes locally on Linux with Microk8s'                                                           
+date: 2019-09-02                                 
+---
+**Authors**: [Ihor Dvoretskyi](https://twitter.com/idvoretskyi), Developer Advocate, Cloud Native Computing Foundation; [Carmine Rimi](https://twitter.com/carminerimi), Product Manager, Canonical
+
+This is article #2 in a series about local deployment options on Linux, and it will cover [MicroK8s](https://microk8s.io/)  - the click-and-run solution for deploying a Kubernetes cluster locally, originally developed by Canonical, the publisher of Ubuntu.
+
+While Minikube usually spins-up a local virtual machine (VM) for the Kubernetes cluster, MicroK8s doesn’t require a VM - it uses [snap](https://snapcraft.io/) packages, an application packaging and isolation technology (also originally developed by Canonical).
+
+This difference has its pros and cons. Here we’ll discuss a few of the interesting differences, comparing the benefits of a VM based approach with the benefits of a non-VM approach. One of the first factors is cross-platform portability. While a Minikube VM is portable across operating systems - it supports not only Linux, but Windows, macOS, and even FreeBSD - Microk8s requires Linux, and only on those distributions [that support snaps](https://snapcraft.io/docs/installing-snapd). Most popular Linux distributions are supported. 
```

Comment:
his difference has its pros and cons. Here we’ll discuss a few of the interesting differences, comparing the benefits of a VM based approach with the benefits of a non-VM approach. 

Should be

his difference has its pros and cons. Here we’ll discuss a few of the interesting differences, and compare the benefits of a VM based approach with the benefits of a non-VM approach.